class/defaults.yml

```diff
@@ -115,7 +124,8 @@ parameters:
         labels: ${keycloak:labels}
         rules: ${keycloak:monitoring:rules}
       postgresql:
-        enabled: ${keycloak:database:builtin}
+        enabled: ${keycloak:database:builtin:enabled}
+        existingSecret: ${keycloak:database:secretname}
```

Comment:
This requires **manually** configuring an additional secret if the builtin database is used. Migration guide doesn't appear to take this into account at the moment, or I overlooked it.

Reply:
It turns out in #12 the default password is simple & hardcoded and not generated. Thus we provide our own secret, which becomes also a bit easier to tell keycloak about this secret (via `helm_values.extraEnvFrom`, which is a string).